source/2019-12-13-countdown-to-the-new-year-ember-simple-auth.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year - Ember Simple Auth
+author: Samanta de Barros
+tags: 2019, Addons, DecEmber
+alias: 2019-12-13-countdown-to-the-new-year-ember-simple-auth.md
+responsive: true
+---
+
+**This is the 13th edition in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 13
+
+Today we're going to look at [ember-simple-auth](https://github.com/simplabs/ember-simple-auth)!
+
+READMORE
+
+### What It Does
+
+The `ember-simple-auth` addon is a lightweight library that helps you implement authentication and authorization in your Ember applications. It takes care of maintaining a client side session and synchronizing it accross tabs and windows, so you don't have to worry about your user signing out in one tab and remaining signed in in others.
+
+It comes with a set of authentication providers (OAuth, Device, Torri) and is easily customizable in case you need to support a different authentication mechanism. It also provides a session service and a series of mixins that will aid you in adding authorization information to your requests, defining which routes should be accessible only to signed in users, and controlling the user session.
```

Comment:
To keep the language linter happy, I think it'd need to skip the word `easy` - what do you think about

```suggestion
It comes with a set of authentication providers (OAuth, Device, Torri) and can be customized in case you need to support a different authentication mechanism. It also provides a session service and a series of mixins that will aid you in adding authorization information to your requests, defining which routes should be accessible only to signed in users, and controlling the user session.
```